src/app/courses/search-courses/courses-search.scss

```
@@ -1,25 +1,58 @@
+        min-height: 36px;
+        padding-top: 4px;
+        padding-bottom: 4px;
+      }
+    }
+  }
+
+  @media (max-width: v.$screen-sm) {
+    grid-template-columns: repeat(2, minmax(0, 1fr));
+    grid-template-rows: repeat(2, minmax(0, 1fr));
+    grid-auto-flow: row;
+    grid-row-gap: 0.5rem;
```

Stylelint (17.14.0)

[error] Expected "grid-row-gap" to be "row-gap" (property-no-deprecated)

(property-no-deprecated)